incubator/kafka/README.md

```diff
@@ -175,3 +187,32 @@ such port at a time, setting the range at every Kafka pod is a reasonably safe c
 * There must not exist a service called `kafka` in the same namespace
 
 [brokerconfigs]: https://kafka.apache.org/documentation/#brokerconfigs
+
+## Prometheus Stats
+
+### Prometheus vs Prometheus Operator
+
+Prometheus is default option of this chart b/c the Prometheus Operator is part of another repo: 
+
+https://github.com/coreos/prometheus-operator/tree/master/helm 
+
+The Prometheus Operator gives additional functionality like automagically updating Prometheus and Alert Manager configuration.  These are things you will
+need to manually do if you are running Prometheus without the Operator.  For more details about the Prometheus Operator:
+
+https://coreos.com/blog/the-prometheus-operator.html
+
+### JMX Exporter
+
+https://github.com/prometheus/jmx_exporter Almost all the stats on Kafka come from JMX.  There are some limited complimentary stats that come from
+Kafka Exporter.  JMX Exporter is a generic exporter not only intended for Kafka.  When using with Kafka you get way more stats than one could want,
```

Comment:
The JMX Exporter is a general purpose prometheus provider which is intended for use with any Java application. Because of this, it produces a number of statistics which may not be of interest. To help in reducing these statistics to their relevant components we have created a curated white list for the JMX exporter. This white list may be modified or removed via the values configuration.